### DISCUSSION TITLE
"Why can't my multi-select tags work? Is it because there's no type casting in the relationship?"

### DISCUSSION THREAD

**Discussion by YANGJIAXUE2022510** [filamentphp/filament#8321] [General]
"After testing, when removing '->relationship', data can be stored in the database using '->multiple()'. Also, using '->options' with '->multiple()' allows data to be stored in the database as well. However, when '->multiple()' is used in conjunction with '->relationship', it results in the following error: 'General error: 1364 Field 'tags' doesn't have a default value.' It seems like data cannot be passed over in this case."
![image](https://github.com/filamentphp/filament/assets/105269516/783e0f49-9a39-4f3e-86c6-13ed9e30f53d)
![39bbaf90a7f7a1c2ca1365d9dff8281](https://github.com/filamentphp/filament/assets/105269516/ef95d970-46ec-484b-8cae-05b9171945cc)
![60ac071af7329cffbcfc43953bd3dec](https://github.com/filamentphp/filament/assets/105269516/e35645db-c5fb-410d-b077-743f7bb6c728)

**Comment by danharrin** [comment]
`->relationship()` writes data to the copywriting_tag pivot table, you need to remove the tags column from the database as it is not used

**Reply by YANGJIAXUE2022510** [reply]
Thank you
